core/src/main/java/org/vertexium/FetchHint.java

```diff
@@ -60,4 +61,18 @@ public static String toString(EnumSet<FetchHint> fetchHints) {
             return EnumSet.copyOf(results);
         }
     }
+
+    public static void checkFetchHints(EnumSet<FetchHint> fetchHints, FetchHint neededFetchHint) {
+        if (!fetchHints.contains(neededFetchHint)) {
```

Comment:
Small change. This method could be implemented as:
```
checkFetchHints(fetchHints, EnumSet.of(neededFetchHint))
```
I'd be fine if they stay the way they are though.